LearningMissionSimulation/PlaygroundVahe.cs

```diff
@@ -156,19 +196,132 @@ List<Module> GetModuleList()
 
         public void CreateClassrooms(int classroomCount)
         {
-            throw new NotImplementedException();
+            ReportHeader("Create classrooms");
+            if (ModuleIdList.Count != 0)
+            {
+                int i = 0;
+                while (i < classroomCount)
+                {
+                    Guid moduleId = ModuleIdList[AttributeGenerator.random.Next(0, ModuleIdList.Count)];
+                    Module module;
+                    ModuleDictionary.TryGetValue(moduleId, out module);
+                    Classroom classroom = ObjectGenerator.GenerateClassroom(module);
+                    ClassroomDictionary.Add(classroom.Id, classroom);
+                    ReportSummary("Classrooms", classroomCount);
+                    ReportFooter("Modules generated");
```

Comment:
Should be
if (ModuleIdList.Count != 0)
{
      .....
}
else
{
     ReportError(ModuleIdList.ToString(), "classroom");
}
 ReportSummary("Classrooms", classroomCount);
 ReportFooter("Modules generated");